dali/python/nvidia/dali/plugin/pytorch/torch_utils.py

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
No test actually checks if the output is correct (or I missed such?). We check shape, device, also we check if the output is deterministic, but we never check if the output is correct. In other words, if Proxy was returning zeros, the tests would still pass.

Reply:
Thanks!